src/azul/service/manifest_service.py

```diff
@@ -419,6 +407,30 @@ def _dss_url(self, file):
         dss_url = config.dss_endpoint + '/' + path
         return dss_url
 
+    @cachedproperty
+    def manifest_content_hash(self) -> int:
+        es_search = self._create_request()
+        es_search.aggs.metric('hash',
```

Comment:
[My suggestion](https://github.com/DataBiosphere/azul/pull/1648#discussion_r398185311) indented differently and I still prefer it that way.

Reply:
And then make all scripts triple-single-quoted strings for symmetry.